techmatters/flex-plugins
# CHI-2358: Add extra line for describing when the status was last updated

## Description

* Add  to the EditSummary modal.
* Use dedicated updateCaseStatus endpoint.

### Checklist

### Related Issues
Fixes #....

### Verification steps

See ticket - requires latest HRM v1.14 & enable_last_case_status_update_info set to true
